### PR TITLE
[FEAT] Decorative Items - Homeless Tent & Nyon Statue

### PR DESCRIPTION
**Pull request by tiffanydys**
# Description
- Added Homeless Tent near Home
- Added Nyon Statue NFT by the river

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
`yarn test`
Screen Shot 2022-04-16 at 5 21 44 PM

`yarn dev`

### Ingredients:
Screen Shot 2022-04-17 at 7 58 51 PM
Screen Shot 2022-04-17 at 7 59 03 PM
Screen Shot 2022-04-17 at 7 59 18 PM

### In-Game:
Screen Shot 2022-04-16 at 4 48 11 PM

Screen Shot 2022-04-16 at 5 14 37 PM

Screen Shot 2022-04-16 at 5 14 01 PM

## Gold Statue Perk: Lore

Screen Shot 2022-04-17 at 9 22 47 PM

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/types/images.ts [hunk 1]
```diff
@@ -50,6 +50,9 @@ import mysteriousParsnip from "assets/nfts/mysterious_parsnip.png";
 import carrotSword from "assets/nfts/carrot_sword.png";
 import nancy from "assets/nfts/nancy.png";
 import kuebiko from "assets/nfts/kuebiko.gif";
+import nyonStatue from "assets/nfts/nyon_statue.png";
+import nyonStatueGold from "assets/nfts/nyon_statue_gold.png";
+import homelessTemt from "assets/nfts/homeless_tent.png";
```
Comment (j923sneaks) [inline src/features/game/types/images.ts:55]: misspell

**Review by Pranav-yadav — CHANGES_REQUESTED** [review]
I would suggest to rename the "Nyon Statue Gold" to **"Golden Nyon Statue"** to keep name consistent with other Golden NFT's, both in code as well as png name 👍

File: src/features/game/types/index.ts [hunk 2]
```diff
@@ -51,6 +51,9 @@ export const KNOWN_IDS: Record<InventoryItemName, number> = {
   "Carrot Sword": 419,
   Nancy: 420,
   Kuebiko: 421,
+  "Nyon Statue": 422,
+  "Nyon Statue Gold": 423,
```
[the remark this answers is not in the document]
Comment (Z1kou) [reply src/features/game/types/index.ts:55]: @Pranav-yadav  Legend

File: src/features/game/types/images.ts (hunk 1 above)
Comment (tiffanydys) [reply src/features/game/types/images.ts]: Corrected. Thanks! :D

File: src/features/game/components/Decorations.tsx [hunk 3]
```diff
@@ -20,6 +20,9 @@ import fountain from "assets/nfts/fountain.gif";
 import beaver from "assets/nfts/beaver.png";
 import apprentice from "assets/nfts/apprentice_beaver.png";
 import foreman from "assets/nfts/construction_beaver.png";
+import nyonStatue from "assets/nfts/nyon_statue.png";
+import nyonStatueGold from "assets/nfts/nyon_statue_gold.png";
```
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/components/Decorations.tsx]: Updated. Thanks! ✌🏻

File: src/features/game/types/craftables.ts [hunk 4]
```diff
@@ -47,7 +47,10 @@ export type BlacksmithItem =
   | "Fountain"
   | "Woody the Beaver"
   | "Apprentice Beaver"
-  | "Foreman Beaver";
+  | "Foreman Beaver"
+  | "Nyon Statue"
+  | "Nyon Statue Gold"
```
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/types/craftables.ts]: Updated. Thanks! ✌🏻

File: src/features/game/types/craftables.ts [hunk 5]
```diff
@@ -312,6 +337,40 @@ export const BLACKSMITH_ITEMS: Record<BlacksmithItem, Craftable> = {
...
+      },
+      {
+        amount: new Decimal(5),
+        item: "Iron",
+      },
+    ],
+    limit: 1,
+    supply: 1000,
+  },
+  "Nyon Statue Gold": {
```
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/types/craftables.ts]: Updated. Thanks! ✌🏻

File: src/features/game/types/images.ts [hunk 6]
```diff
@@ -50,6 +50,9 @@ import mysteriousParsnip from "assets/nfts/mysterious_parsnip.png";
 import carrotSword from "assets/nfts/carrot_sword.png";
 import nancy from "assets/nfts/nancy.png";
 import kuebiko from "assets/nfts/kuebiko.gif";
+import nyonStatue from "assets/nfts/nyon_statue.png";
+import nyonStatueGold from "assets/nfts/nyon_statue_gold.png";
```
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/types/images.ts]: Updated. Thanks! ✌🏻

File: src/features/game/types/images.ts [hunk 7]
```diff
@@ -390,6 +393,18 @@ export const ITEM_DETAILS: Items = {
     ...MARKET_ITEMS["Carrot Sword"],
     image: carrotSword,
   },
+  "Nyon Statue": {
+    ...BLACKSMITH_ITEMS["Nyon Statue"],
+    image: nyonStatue,
+  },
+  "Nyon Statue Gold": {
```
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/types/images.ts]: Updated. Thanks! ✌🏻

File: src/features/game/types/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (tiffanydys) [reply src/features/game/types/index.ts]: Updated. Thanks! ✌🏻

**Comment by adamhannigan** [comment]
Thanks @tiffanydys 

Just a heads up, we will keep this PR open until we can build the backend/smart contract details

**Comment by adamhannigan** [comment]
@tiffanydys Awesome work 👏 

For now we have just included one of the Nyon Statues. We have to be careful releasing too many decorative items. We want to release more NFT Buffs before we do more decorations 🤗 

Also, I noticed you manually created the markdown in the public/erc1155 folder.

We also have a `/markdown` folder. You can enter the details in a markdown file (e.g. `423.md`) and then run `yarn erc1155` to automatically update our JSON files 😄

**Comment by tiffanydys** [comment]
> @tiffanydys Awesome work 👏 
> 
> 
> 
> For now we have just included one of the Nyon Statues. We have to be careful releasing too many decorative items. We want to release more NFT Buffs before we do more decorations 🤗 
> 
> 
> 
> Also, I noticed you manually created the markdown in the public/erc1155 folder.
> 
> 
> 
> We also have a `/markdown` folder. You can enter the details in a markdown file (e.g. `423.md`) and then run `yarn erc1155` to automatically update our JSON files 😄 

I didn't know we had that 👀 honestly was having a hard time editing it in one line 😅 should have figured it out that if it doesn't feel right then i'm probably doing it the wrong way lol. 
Thank you for the guidance! ❤️🤗

**Pull request merged**